test/basictests.jl

```diff
@@ -161,6 +161,10 @@ sol = benchmark_scalar(sf, csu0)
 @test sol.retcode === ReturnCode.Success
 @test sol.u * sol.u - 2 < 1e-9
 
+@test (@ballocated benchmark_immutable(ff, cu0)) < 400
+@test (@ballocated benchmark_mutable(ff, cu0)) < 400
```

Comment:
the allocation test we should check on this one shouldn't include the problem building.